app/src/main/java/chat/rocket/android/RocketChatApplication.java

```diff
@@ -23,6 +23,7 @@
   public void onCreate() {
     if (BuildConfig.DEBUG) {
```

Comment:
I believe you could change something on `circle.yml` file that will run the code in debug mode.